### DISCUSSION TITLE
Is there a way to stop calling `eth_getBalance` when rendering RainbowKitProvider

### DISCUSSION THREAD

**Discussion by einaralex** [rainbow-me/rainbowkit#1479] [Q&A] [answered]
My dapp does not need the balance of the native currency and I've made sure that `showBalance` on the ConnectButton is off, but RainbowKitProvider still calls `eth_getBalance` every time it's rendered, is there a way to stop this?

**Comment by DanielSinclair** [comment, marked as answer]
Not currently, but this is a good find and we'll get it resolved. Created #1488 to track the progress on the fix.
